URBNSwiftAlert/Classes/AlertStyler.swift

```diff
@@ -80,17 +80,22 @@ extension AlertStyler {
          * Pass no to disable blurring in the background
          */
         public var isEnabled = true
+        
+        /**
+         * Tint color of the blur.  If Blur.isEnabled = false, then this is the color of the AlertViewController.view
+         */
+        public var tint: UIColor = .clear {
+            didSet {
+                assert(tint.cgColor.alpha < 1.0 && isEnabled, "URBNAlertStyle: blurTintColor alpha component must be less than 1.0 to see the blur effect. Please use colorWithAlphaComponent: when setting a custom blurTintColor, for example: UIColor.white.withAlphaComponent(0.4)")
```

Comment:
Don’t assert in this case. Rather set `tint = 1` if it’s >1 and `tint = 0` if it’s < 0. If you look at UIColor, this is what it does:

```swift
let color = UIColor(red: 1.0, green: 1.0, blue: 0.0, alpha: 0.3)
color.withAlphaComponent(5.0) // returns alpha 1.0
color.withAlphaComponent(-42) // returns alpha 0.0
```

The reason why is because that color could be computed. With float computations, it could inadvertently be slightly > 1 or < 0